test/unit/spec/ForgotPassword_spec.js

```diff
@@ -296,6 +296,28 @@ function (Q, Okta, OktaAuth, Util, AccountRecoveryForm, PrimaryAuthForm, Beacon,
             });
           });
       });
+      itp('should correct show back link in PwdResetEmailSent page', function () {
+        return setup({'features.hideBackToSignInForReset': true}).then(function (test) {
+          test.setNextResponse(resChallengeEmail);
+          test.form.setUsername('foo');
+          test.form.pressEnter();
+          return Expect.waitForPwdResetEmailSent(test);
+        })
+          .then(function (test) {
+            expect(test.form.backToLoginButton().length).toBe(0);
+          });
+      });
+      itp('should correct show hide link in PwdResetEmailSent page', function () {
```

Comment:
`should show back link in PwdResetEmailSent page when hideBackToSignInForReset is false`